app/pages/faq.cjsx

```diff
@@ -0,0 +1,24 @@
+counterpart = require 'counterpart'
+React = require 'react'
+{Markdown} = (require 'markdownz').default
+
+counterpart.registerTranslations 'en',
+  faqPage:
+    content: '''
+
+      ### Frequently Asked Questions
+
+      - **What browser version does Zooniverse support?**
+        We support major browsers up to the second to last version.
```

Comment:
I think technically Edge and IE are technically separate browsers since what they support is very different with Edge now supporting web standards. I'm hoping, however, we aren't going to support IE 11 too much longer as people upgrade to Windows 10/Edge. I wouldn't put specific version numbers because of the rapid release schedule both Chrome and Firefox do, so this could quickly get out of date.